src/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace GuzzleHttp\Exception;
+
+class InvalidArgumentException extends \InvalidArgumentException implements GuzzleException {}
```

Comment:
I think this class should be final for now, we can always open it up later.

Also, please add a trailing new line to the end of the file.

Reply:
Done.. I added the new line but I am not sure if it is visible through the PR changes tab